apps/teamsjs-diagnostic-app/src/index.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { createRoot } from "react-dom/client";
+import App from "./components/App";
+import "./index.css";
+
+const container = document.getElementById("root");
+const root = createRoot(container!);
```

Comment:
Another question, what was the thought process behind the choosing react version?

Reply:
I chose React because of its rendering capabilities which can help my diagnostic app handle complex UI features, which is crucial since I will be doing dynamic API interactions and log management. Also, React is useful since I am sideloading onto Microsoft Teams, which helps me when supporting various TeamsJS versions and creating a good user experience across different Teams environments.